portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,496 @@
+    align-items: center;
+    float: right;
+    position: relative;
+}
+nav ul{
+    display: flex;
+}
+nav a{
+    color: #fff;
+    font-size: 1.5rem;
+}
+.container {
+    width: auto;
+    max-width: 120rem; 
+    height: 100%; 
+    margin: 0 auto; 
+    position: relative; 
+    padding: 0 100px 0 100px;
+}
+.section-heading{
+    text-align: center;
+    margin: 0;
+}
+.section-heading h1{
+    font-size: 3.5rem;
+    color: rgba(255, 255, 255, .7);
+    text-transform: uppercase;
+    font-weight: 300;
+    position: relative;
+    margin-bottom: 1rem;
+}
+.section-heading h1::before,
+.section-heading h1::after{
+    content: '';
+    position: absolute;
+    bottom: -.5rem;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: #EC361B;
+}
+.section-heading h1::before{
+    width: 10rem;
+    height: 3px;
+    border-radius: .8rem;
+}
+.section-heading h1::after{
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 100%;
+    bottom: -1rem;
+}
+.section-heading h3{
+    font-weight: 300;
+    margin-bottom: 1rem;
+    position: relative;
+    font-size: 1.5em;
+    text-align: left;
+}
+.section-heading h6{
+    font-size: 1.6rem;
+    font-weight: bold;
+    margin-bottom: 2rem;
 }
-
 #greeting-container {
   margin-top: 20px;
+  font-size: 16px;
+}
+section{
+    padding: 6rem 0;
+}
+.text {
+    margin-top: 120px;
+}
+/* .skills h1,
+.projects h1,
+.experience h1,
+.contact h1{
+    color: #fff;
+} */
+li{
+    list-style: none;
+    margin-left: 30px;
+}
+.about .container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    /* padding: 0; */
+}
+.about-heading{
+    text-align: center;
+    text-transform: uppercase;
+    line-height: 0;
+    margin-bottom: 6rem;
+}
+.about-heading h1{
+    font-size: 10rem;
+    opacity: .3;
+    color: #EC361B;
+}
+.about-heading h6{
+    font-size: 2rem;
+    font-weight: 300;
+}
+.profile-img{
+    flex: 1;
+    margin-right: 5rem;
+    background-image: linear-gradient(50deg, #ec7d1b, #EC361B);
+    z-index: -1;
+}
+.about-details{
+    flex: 1;
+}
+.social-media{
+    margin-top: 5rem;
+}
+.interest-list{
+    font-size: 15px;
+    font-weight: 300;
+}
+.social-media i{
+    font-size: 4rem;
+    color: rgb(255, 255, 255);
+    transition: color 650ms;
+}
+.social-media i:hover {
+    color: #EC361B;
+}
+.nav-list{
+    list-style: none;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.skills{
+    padding: 0%;
+}
+.myskills{
+    width: 500px;
+    margin: 60px auto;
+    color: white;
+}
+.myskills li{
+    margin: 20px 0;
+}
+.bar{
+    position: relative;
+    display: block;
+    height: 12px;
+    border: 1px solid rgba(3, 3, 3, 0.5);
+    border-radius: 3px;
+    overflow: hidden;
+    box-shadow: 0 0 10px #D0D0D0;
+}
+.bar span{
+    height: 12px;
+    float: left;
+    background-image: linear-gradient(50deg, #D0D0D0, #B0B0B0);
+}
+.html{
+    width: 50%;
```

Comment:
It would be cool to make percentages driven by server response. For example server returns JSON with list of languages and percentages and client shows them as animated bars.

Just an idea...

Reply:
Yes that would definitely be way cooler, I will try to implement it. Thank you!

Reply:
Just keep the though in mind. As you explore different technologies you can notice some pieces that can be used for the goal. Even if you never implement the feature you'll still notice how pieces of technology can be used.